apps/src/p5lab/spritelab/blocks.js

```diff
@@ -104,38 +105,52 @@ const customInputTypes = {
   locationPicker: {
     addInput(blockly, block, inputConfig, currentInputRow) {
       currentInputRow.appendField(
-        `${inputConfig.label}(0, 0)`,
+        `${inputConfig.label}`,
         `${inputConfig.name}_LABEL`
       );
       const fieldRow = currentInputRow.getFieldRow();
-      const label = fieldRow[fieldRow.length - 1];
+      const fieldLabel = fieldRow[fieldRow.length - 1]; // fieldLabel = 'sprite at'
```

Comment:
Is this comment intentional? I'm not quite sure what it means.

Reply:
It is what is assigned to `fieldLabel` at this point. It was more of a note to myself and I forgot to remove - thanks!